Homework_1.py

```diff
@@ -0,0 +1,97 @@
+# Задача 2: Найдите сумму цифр трехзначного числа. 
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+
+'''
+num = int(input('Введите 3-х значное число: '))
+
+if num <100 or num > 999:
+    print('Число не 3-х значное')
+else:
+    print((num % 10) + ((num % 100) // 10) + (num // 100))
```

Comment:
Тут скобки не нужны. Приоритет операций стандартный.
```suggestion
    print(num % 10 + num % 100 // 10 + num // 100)
```